src/web/components/StickyBottomBanner/StickyBottomBanner.tsx

```diff
@@ -117,11 +117,41 @@ export const StickyBottomBanner = ({
             setSelectedBanner(PickedBanner),
         );
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [isSignedIn, asyncCountryCode, asyncBrazeUuid]);
+    }, []); // Empty dependency array because we only want this to run once
 
     if (SelectedBanner) {
         return <SelectedBanner />;
     }
 
     return null;
 };
+
+// This outer component exists because we don't want to run the banner picker
+// until all of our dependencies are defined. Then when they are all defined we
+// only want to run the banner picker once.
+export const StickyBottomBanner = ({
+    isSignedIn,
+    asyncCountryCode,
+    CAPI,
+    asyncBrazeUuid,
+    isDigitalSubscriber,
+}: Props) => {
+    if (
```

Comment:
Is this just defensive in purpose?

Reply:
Yeah exactly. All of those props will be set to _something_ but not necessarily on the first render cycle (because they're set via effects in the parent component). So we're making sure that the banner picker doesn't run until we've got all the inputs we need as we only want it to run once per page.